keyboards/reviung/reviung39/keymaps/junonum/keymap.c

```diff
@@ -0,0 +1 @@
+#include "layouts/community/ortho_4x12/junonum/keymap.c"
```

Comment:
All `.c/h` files need a license header.
eg.
```
// Copyright 2023 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
You shouldn't be including c files like this.

Reply:
@drashna I'm aware that this is not the orthodox way to do things, but neither is what I'm trying to do.

What I'm trying to do is:
* Use my community keymap `ortho_4x12/junonum` on `reviung39` .
* This keymap however requires mapping the `reviung39` to `LAYOUT_ortho_4x12`
* [My previous PR, attempting to do so, was rejected, and I was told to settle this in an userspace](https://github.com/qmk/qmk_firmware/pull/19733#event-8416143700)
* What I currently have is the outcome of some very long conversations on the QMK discord channel with some of the core maintainers. It was pretty clear that what I asked for was not standard, and neither was the suggestion I was provided with.

If there is a neat and standard-compliant way to do this that I'm missing out, I would be very elated hear about it and make the changes accordingly.

Reply:
@drashna Let me know if this is actually not possible in the meantime. I don't want my un-merge-able PR to bloat up your works (that being said, I would definitely appreciate if this can be looked into in the future).